cypress.config.js

```diff
@@ -2,13 +2,18 @@ const { defineConfig } = require('cypress');
 const preprocessor = require('@badeball/cypress-cucumber-preprocessor');
 const browserify = require('@badeball/cypress-cucumber-preprocessor/browserify');
 
-async function setupNodeEvents(on, config) {
-  await preprocessor.addCucumberPreprocessorPlugin(on, config);
+/**
+ * @param {object} events - Cypress.PluginEvents.
+ * @param {object} configuration - Cypress.PluginConfigOptions.
+ * @returns {object} - Config as it might have been modified by the plugin.
```

Comment:
async function -> returns Promise

Reply:
done

Reply:
@returns {Promise<object>} Promise with config as it might have been modified by the plugin on success otherwise an error.
(std pattern for async functions)